From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Fri, 18 Sep 2020 18:09:23 +0200
Subject: Avoid privacy breach in HTML

Origin: Debian
Forwarded: no
Last-Update: 2020-02-24

Use html5shiv-printshiv from the Debian node-html5shive package
Last-Update: 2020-02-24
---
 doc/8.topics/compander-limiter.htm | 2 +-
 doc/8.topics/slop-tilde.htm        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/8.topics/compander-limiter.htm b/doc/8.topics/compander-limiter.htm
index d50f36e..6d25a2f 100644
--- a/doc/8.topics/compander-limiter.htm
+++ b/doc/8.topics/compander-limiter.htm
@@ -12,7 +12,7 @@
       div.column{display: inline-block; vertical-align: top; width: 50%;}
   </style>
   <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
+    <script src="file:///usr/share/nodejs/html5shiv/dist/html5shiv-printshiv.min.js"></script>
   <![endif]-->
 </head>
 <body>
diff --git a/doc/8.topics/slop-tilde.htm b/doc/8.topics/slop-tilde.htm
index db9ba91..26c18aa 100644
--- a/doc/8.topics/slop-tilde.htm
+++ b/doc/8.topics/slop-tilde.htm
@@ -12,7 +12,7 @@
       div.column{display: inline-block; vertical-align: top; width: 50%;}
   </style>
   <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
+    <script src="file:///usr/share/nodejs/html5shiv/dist/html5shiv-printshiv.min.js"></script>
   <![endif]-->
 </head>
 <body>
